node-fs-writefile/cp.js

```diff
@@ -0,0 +1,8 @@
+import { writeFile, readFile } from 'node:fs/promises';
+try {
+  const [, , ...fileList] = process.argv;
+  const data = await readFile(fileList[0], 'utf-8');
+  await writeFile(fileList[1], data, 'utf8');
+} catch (err) {
+  console.log('Error: ', err.message);
```

Comment:
You usually want to log the entire error, not just the message field. That way you will get a stack trace that will help you figure out where things went wrong.